tox.ini

```diff
@@ -5,10 +5,23 @@ envlist = py36
 commands =
     flake8 reconcile tools e2e_tests
     pylint -j0 reconcile tools e2e_tests
-    pytest
+    pytest --cov=reconcile
```

Comment:
let's also add `--cov-report=term-missing`. That's shows the gaps.

Reply:
Fixed!